libcxx/include/__chrono/day.h

```diff
@@ -92,6 +93,17 @@ _LIBCPP_HIDE_FROM_ABI inline constexpr day& day::operator-=(const days& __dd) no
 
 } // namespace chrono
 
+#  if _LIBCPP_STD_VER >= 26
+
+template <>
+struct hash<chrono::day> {
+  _LIBCPP_HIDE_FROM_ABI static size_t operator()(const chrono::day& __d) noexcept {
```

Comment:
Test failures indicated that we should include `<__cstddef/size_t.h>` in every header where `hash` specialization is being added.